scripts/cem-plugin-react-wrapper.js

```diff
@@ -0,0 +1,277 @@
+        async packageLinkPhase({ customElementsManifest }) {
+            const { name: pkgName, version: pkgVersion } = JSON.parse(
+                await readFile(`${process.cwd()}/package.json`)
+            );
+            const { modules } = customElementsManifest;
+            const declMap = new Map(
+                modules.flatMap((m) =>
+                    m.declarations.map((decl) => [decl.name, decl])
+                )
+            );
+            const components = modules.flatMap((m) =>
+                m.declarations.filter(
+                    (decl) =>
+                        !exclude.includes(decl.name) &&
+                        (decl.customElement || decl.tagName)
+                )
+            );
+
+            if (components.length === 0) {
+                return;
+            }
+
+            const componentImports = [];
+
+            const fileImports = modules
+                .filter(
+                    (m) =>
+                        m.declarations.length === 0 &&
+                        m.path.indexOf('sync/') === -1
+                )
+                .map(
+                    (m) =>
+                        `import '${pkgName}/${m.path?.replace('.ts', '.js')}';`
+                );
+            if (pkgName === '@spectrum-web-components/theme') {
+                fileImports.push(
+                    "import '@spectrum-web-components/theme/theme-darkest.js';"
+                );
+                fileImports.push(
+                    "import '@spectrum-web-components/theme/theme-dark.js';"
+                );
+                fileImports.push(
+                    "import '@spectrum-web-components/theme/theme-light.js';"
+                );
+                fileImports.push(
+                    "import '@spectrum-web-components/theme/theme-lightest.js';"
+                );
+                fileImports.push(
+                    "import '@spectrum-web-components/theme/scale-medium.js';"
+                );
+                fileImports.push(
+                    "import '@spectrum-web-components/theme/scale-large.js';"
+                );
+                fileImports.push(
+                    "import '@spectrum-web-components/theme/express/theme-darkest.js';"
+                );
+                fileImports.push(
+                    "import '@spectrum-web-components/theme/express/theme-dark.js';"
+                );
+                fileImports.push(
+                    "import '@spectrum-web-components/theme/express/theme-light.js';"
+                );
+                fileImports.push(
+                    "import '@spectrum-web-components/theme/express/theme-lightest.js';"
+                );
+                fileImports.push(
+                    "import '@spectrum-web-components/theme/express/scale-medium.js';"
+                );
+                fileImports.push(
+                    "import '@spectrum-web-components/theme/express/scale-large.js';"
+                );
+            }
```

Comment:
The above imports are side effectful and as such can't be tree shaken when unused by an app, do you have any other thoughts on how to leverage them here. We could create reexporting files, document using this in collaboration with `@spectrum-web-components/theme`, something else?

Reply:
The reason that I hardcoded these import statements is that the `customer-elements.json` file does not provide these files' information. But I do agree with you about tree shaking. Is it possible to have the user explicitly declare it in their own code? For example:

```
import "./App.css";

import { Theme } from "@swc-react/theme";
import { Button } from "@swc-react/button";
import { Card } from "@swc-react/card";
import { Radio, RadioGroup } from "@swc-react/radio";
import { ActionGroup } from "@swc-react/action-group";
import { ActionButton } from "@swc-react/action-button";
import { ProgressCircle } from "@swc-react/progress-circle";
import { IconEdit } from "@swc-react/icons-workflow/Edit";
import '@spectrum-web-components/theme/theme-light.js';
import '@spectrum-web-components/theme/scale-medium.js';

function App() {
  return (
    <Theme theme="spectrum" scale="medium" color="light">
      <div className="App">
        <Button click={() => alert("Hello")}>
          <IconEdit slot="icon"></IconEdit>Be right back
        </Button>
        <Card heading="Card Heading" subheading="JPG Photo">
          <img
            slot="cover-photo"
            src="https://picsum.photos/200/300"
            alt="demo"
          />
          <div slot="footer">Footer</div>
        </Card>
        <RadioGroup selected="first" name="example">
          <Radio value="first">Option 1</Radio>
          <Radio value="second">Option 2</Radio>
          <Radio value="third">Option 3</Radio>
          <Radio value="fourth">Option 4</Radio>
        </RadioGroup>
        <ActionGroup selects="single" selected={["third"]}>
          <ActionButton value="first">First</ActionButton>
          <ActionButton value="second">Second</ActionButton>
          <ActionButton value="third">Third</ActionButton>
        </ActionGroup>
        <ProgressCircle size="xxl" indeterminate />
      </div>
    </Theme>
  );
}

export default App;

```

Reply:
They certainly can, and it aligns more closely with the "native" usage. Let's go with this approach!